src/article/shared/FootnoteComponent.js

```diff
@@ -37,6 +39,22 @@ export default class FootnoteComponent extends Component {
         )
       )
     }
+
+    if (mode === METADATA_MODE) {
+      const Button = this.getComponent('button')
+      const footer = $$('div').addClass('se-footer').append(
+        $$(Button).addClass('se-remove-item').append(
```

Comment:
Try to use as much of the component props as u need, e.g. in this case you could provide icon, and theme.
https://github.com/substance/texture/blob/master/src/kit/ui/Button.js#L3-L25